Align POPRF issuance with the latest VOPRF draft (#91)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-protocol.md

Both Client and Issuer also share a common public string called `info`.

Given this configuration and these inputs, the two messages exchanged in
this protocol are described below. This section uses notation described in
{{OPRF, Section 4}}, including SerializeElement and DeserializeElement,
SerializeScalar and DeserializeScalar, and DeriveKeyPair.

## Client-to-Issuer Request {#client-to-issuer}

> The Client first creates an OPRF context as follows:

Reviewer: ```suggestion
The Client first creates a context as follows:
```
